README.md

```diff
@@ -1,6 +1,6 @@
 # Prometheus Basics Training
 
-Interactive Prometheus Basics Training: [prometheus-training.puzzle.ch](https://prometheus-training.puzzle.ch/)
+In this guided hands-on training, we show the participants the basics of Prometheus, the package manager for Kubernetes.
```

Comment:
The last part of the sentence should be changed to something like "an open-source monitoring solution."

Reply:
https://github.com/puzzle/prometheus-training/pull/369